src/content/docs/en/guides/cms/sitecore.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Sitecore Experience Manager & Astro
+description: Add content to your project using Sitecore as your CMS.
+type: cms
+stub: true
+service: Sitecore XM
+i18nReady: false
+---
+
+[Sitecore Experience Manager™ (XM)](https://www.sitecore.com/products/experience-manager) is a powerful and intuitive CMS for the simplified creation and management of experiences across channels and devices.
+
+## Getting started
+
+1. Create a Sitecore Headless website following official [documentation](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html).
+2. Run the project initialization command in your terminal: 
+  ```shell
+  npx @astro-sitecore-jss/create-astro-sitecore-jss@latest
+  ```
+3. Follow the instructions in the terminal to create your project.
+4. Start Astro website:
+  ```shell
+  npm run dev
+  ```
```

Comment:
```suggestion
```
Starting the dev server doesn't feel like a required step to connecting your project, only something you'd do when you're ready to start developing, so I think we can remove!